src/Components/Form/Form.js

```diff
@@ -3,24 +3,45 @@ import Select from 'react-select';
 import transportOptions from '../../assets/travelModeData';
 import TripETA from '../TripETA/TripETA';
 import Autocomplete from 'react-google-autocomplete';
-
+import { gql, useMutation } from '@apollo/client';
 // import {SearchLocationInput} from '../SearchLocationInput/SearchLocationInput.js'
 // import {SearchLocationInput2} from '../SearchLocationInput2/SearchLocationInput2.js'
 // import TripDuration from '../TripDuration/TripDuration';
 
+const CREATE_TRIP = gql `
+  mutation CreateTrip($startPoint: String!, $endPoint: String!, $travelMode: String!){
```

Comment:
I'm guessing we can skip propTypes, then, if we're typechecking right in the mutation?